tests/comfy_cli/cql/test_proxy_migration.py

```
@@ -0,0 +1,407 @@
+    assert promoted.next_unique_name("seed", ["seed"]) == "seed_1"
+    assert promoted.next_unique_name("seed", ["seed", "seed_1"]) == "seed_2"
+    # the counter restarts from the BASE name, never from an existing suffix
+    assert promoted.next_unique_name("seed", ["seed", "seed_2"]) == "seed_1"
+
+
+# --------------------------------------------------------------------------- #
+# classification (``classify`` in the frontend)
+# --------------------------------------------------------------------------- #
+
+
+def test_flux_plan_classifies_linked_repairable_and_unrepairable(graph):
```

Pylint (4.0.7)

[convention] Missing function or method docstring

(C0116)